### PR TITLE
Improvements to examples and Lynx GH action

### PR DESCRIPTION
**Pull request by cygnusv**
**Type of PR:**

- [X] Bugfix
- [ ] Feature
- [ ] Documentation
- [X] Other

**Required reviews:**

- [ ] 1
- [X] 2
- [ ] 3

**What this does:**
* Fixes nodejs script so exceptions are not catched, particularly because we use this as a canary with GH actions.
* Changes to nodejs script to reflect that Creator and Consumer are different entities (currently it was the same address, which is confusing and misleading)
* Update domain usage in demos and examples

**Issues fixed/closed:**
> - Fixes #...

**Why it's needed:**
> Explain how this PR fits in the greater context of the NuCypher Network.
> E.g., if this PR address a `nucypher/productdev` issue, let reviewers know!

**Notes for reviewers:**
> What should reviewers focus on?
> Is there a particular commit/function/section of your PR that requires more attention from reviewers?

### CODE REVIEW

**Comment by piotr-roslaniec** [comment]
LGTM. I will contribute changes to other examples in a separate PR.

File: examples/taco/nodejs/.env.example [hunk 1]
```diff
@@ -1,2 +1,6 @@
-RPC_PROVIDER_URL=https://polygon-mumbai.infura.io/v3/3747007a284045d483c342fb39889a30
-PRIVATE_KEY=0x900edb9e8214b2353f82aa195e915128f419a92cfb8bbc0f4784f10ef4112b86
+# URL for RPC blockchain provider. For TACo testnet you should use a Polygon Mumbai endpoint.
+RPC_PROVIDER_URL=
+# Hex-encoded private keys for the Creator and the Consumer.
+# We provide here some defaults, but we encourage you to choose your own.
+CREATOR_PRIVATE_KEY=0x900edb9e8214b2353f82aa195e915128f419a92cfb8bbc0f4784f10ef4112b86
```
Comment (derekpierre) [inline examples/taco/nodejs/.env.example:5]: Minor nitpick - what do you think about:
creator -> encryptor

**Review by derekpierre — APPROVED** [review]
🎸  - Looks great! One minor nitpicky suggestion - but up to you.

**Review by manumonti — APPROVED** [review]
LGTM! ✌️

File: examples/taco/nodejs/.env.example (hunk 1 above)
Comment (cygnusv) [reply examples/taco/nodejs/.env.example]: ✅  see 1205204b7a9a4c2617590e8c69e0a0cd3c9f8502

File: examples/taco/nodejs/.env.example [hunk 2]
```diff
@@ -1,5 +1,6 @@
 # URL for RPC blockchain provider. For TACo testnet you should use a Polygon Mumbai endpoint.
-RPC_PROVIDER_URL=
+# Current default extracted from https://chainlist.org/chain/80001, but ideally, you'll use your own endpoint.
+RPC_PROVIDER_URL=https://polygon-mumbai-bor.publicnode.com
```
Comment (derekpierre) [inline examples/taco/nodejs/.env.example:3]: 💥

**Pull request merged**